File: hw/Rodriguez/u2/hw18/Zoo/ec_edu_espe_zoo_model/feline.py
from .mammal import Mammal
from datetime import date
from .cage import Cage

class Feline(Mammal):
    def __init__(self, id, description, date_onborn, gender, number_of_bones, cage):
        super().__init__(id, description, date_onborn, gender, number_of_bones, cage)

File: ws/ccedeno/u2/WS24Zoo/Zoo/src/main/java/ec/edu/espe/zoo/model/Feline.java
package ec.edu.espe.zoo.model;

import java.util.Date;

/**
 *
 * @author Andres Cedeno, codecreators, DCCO-ESPE
 */
public abstract class Feline extends Mammal{

    public Feline(int id, String description, Date dateOnborn, char gender, int numberOfBones, Cage cage) {
        super(id, description, dateOnborn, gender, numberOfBones, cage);
    }

}
